src/lib.rs

```diff
@@ -154,6 +154,13 @@ pub struct PushConstantInfo {
     pub size: u32,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct EntryPoint {
+    pub execution_model: spirv::ExecutionModel,
+    pub id: u32,
```

Comment:
The type of this number is indeed an `IdRef` operand, but the name should refect what it's pointing to: the `OpFunction` that this `OpEntryPoint` is describing. Can you rename this to `function_id`?